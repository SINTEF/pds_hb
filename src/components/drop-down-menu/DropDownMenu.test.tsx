import React from 'react'
import renderer from 'react-test-renderer'
import { DropDownMenu } from '.'

it('renders company user without crashing', () => {
  const tree = renderer
    .create(<DropDownMenu username="Jens Jensen" isCompanyUser={true} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders non-company user without crashing', () => {
  const tree = renderer
    .create(<DropDownMenu username="Jens Jensen" isCompanyUser={false} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
