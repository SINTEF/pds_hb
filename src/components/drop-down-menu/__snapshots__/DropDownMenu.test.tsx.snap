// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders company user without crashing 1`] = `
<div>
  <div
    className="title notClicked"
  >
    <div
      className="usersymbol"
      onClick={[Function]}
    >
      J
    </div>
  </div>
</div>
`;

exports[`renders non-company user without crashing 1`] = `
<div>
  <div
    className="title notClicked"
  >
    <div
      className="usersymbol"
      onClick={[Function]}
    >
      J
    </div>
  </div>
</div>
`;
